kube/pod.go

```diff
@@ -167,14 +168,37 @@ func getVolumeMounts(role *model.Role) helm.Node {
 	return helm.NewNodeList(mounts...)
 }
 
-func getEnvVars(role *model.Role, defaults map[string]string, secrets SecretRefMap, createHelmChart bool) (helm.Node, error) {
+func getEnvVars(role *model.Role, defaults map[string]string, secrets SecretRefMap, settings *ExportSettings) (helm.Node, error) {
 	configs, err := role.GetVariablesForRole()
 	if err != nil {
 		return nil, err
 	}
 
+	re := regexp.MustCompile("^KUBE_SIZING_([A-Z][A-Z_]*)_COUNT$")
```

Comment:
I think the point is that all roles that depend on this information should be re-deployed by helm automatically when the replica counts of those roles change. So even if you don't use the values, you still want to reference the names in your template variables to force the update.

This is similar to `((#MYSQL_HCF_CLUSTER_CONFIG_REVISION))((/MYSQL_HCF_CLUSTER_CONFIG_REVISION))` under HCP, except more fine-grained.

And once you have those variables, you might as well use them instead of talking to the kube API.

Reply:
Ah, okay, that makes sense